### PR TITLE
Switched bloop back to scalacenter/bloop

### PR DESCRIPTION
**Pull request by KristianAN**
Preparing to switch back to scalacenter/bloop so we can start merging in the pending PRs.

### CODE REVIEW

**Comment by KristianAN** [comment]
@oyvindberg Maybe you have some insight into why the windows build fails?

**Comment by oyvindberg** [comment]
No, unfortunately. Im considering disabling the windows build soon, until a motivated windows user comes along. I have no idea whether the windows port actually works now before this PR.

File: bleep-core/src/scala/bleep/bsp/SetupBloopRifle.scala
Comment (oyvindberg) [inline bleep-core/src/scala/bleep/bsp/SetupBloopRifle.scala:44]: They only publish it for 2.12?!

**Review by oyvindberg — APPROVED** [review]
Nice! Will merge when we get it green

**Comment by oyvindberg** [comment]
I'll try to do some debug magic tomorrow or Sunday night if you can't find a way for windows.

File: bleep-core/src/scala/bleep/bsp/SetupBloopRifle.scala
Comment (KristianAN) [reply bleep-core/src/scala/bleep/bsp/SetupBloopRifle.scala:44]: It would seem so. I'm not really sure why. Probably it just has not been a priority while the fork was maintained?

**Comment by KristianAN** [comment]
> I'll try to do some debug magic tomorrow or Sunday night if you can't find a way for windows.

I would not even know where to start as I don't have access to a windows machine. It's worth noting that the windows CI seems to be very flaky for scalacenter/bloop as well. 

Examples:
https://github.com/scalacenter/bloop/actions/runs/9900282981
https://github.com/scalacenter/bloop/actions/runs/9893468259
https://github.com/scalacenter/bloop/actions/runs/9892903253

File: bleep-core/src/scala/bleep/bsp/SetupBloopRifle.scala
Comment (sideeffffect) [reply bleep-core/src/scala/bleep/bsp/SetupBloopRifle.scala:44]: > They only publish it for 2.12?!

Is that really so @tgodzik?

Also, why are you using `bloop-frontend` when scala-cli is using `bloop-rifle`?

Relevant PRs
 * https://github.com/scalacenter/bloop/pull/2355
 * https://github.com/VirtusLab/scala-cli/pull/3020

Comment (tgodzik) [reply bleep-core/src/scala/bleep/bsp/SetupBloopRifle.scala:44]: But looking here, I think that's fine. It's just the internal version of the server. I think we might be able to update to 2.13 now, but didn't have the chance to look.

Comment (oyvindberg) [reply bleep-core/src/scala/bleep/bsp/SetupBloopRifle.scala:44]: No idea where the frontend dependency comes from, it should all be rifle, just like scala-cli

Comment (tgodzik) [reply bleep-core/src/scala/bleep/bsp/SetupBloopRifle.scala:44]: That's the dependency for starting Bloop server itself. Bloop rifle doesn't download anything by itself.

Comment (KristianAN) [reply bleep-core/src/scala/bleep/bsp/SetupBloopRifle.scala:44]: This is because bleep bundles bloop instead of expecting the user to have it on PATH right?

Comment (KristianAN) [reply bleep-core/src/scala/bleep/bsp/SetupBloopRifle.scala:44]: Doesn't scala-cli and metals have to do something similar? (metals might not need to as it can depend on the build tool to manage it). If the user does not have bloop on PATH you will need bloop-frontend to start the server, no?

Comment (tgodzik) [reply bleep-core/src/scala/bleep/bsp/SetupBloopRifle.scala:44]: We never look for Bloop on PATH. If it was already started then we can just use the same artifacts it downloaded, otherwise we just download what is needed and provide classpath to bloop rifle. But yes, the current approach here is the same as Scala CLI and Metals do it.

**Review by tgodzik — APPROVED** [review]
From my perspective everything here is expected, not sure about the windows issues. The ones in Bloop are normally related to DAP timing out.

**Comment by oyvindberg** [comment]
tried some cleanups of earlier hacks to get the windows build going. so far we're blocked on this thing:
```
📙 Failed to delete C:\Users\runneradmin\AppData\Local\bleep\cache\socket\proc-196 at shutdown: C:\Users\runneradmin\AppData\Local\bleep\cache\socket\proc-196\output: The process cannot access the file because it is being used by another process
```

I think I struggled with this the last update as well. the problem is fundamentally that we're building bleep with and old bloop, then using the new bleep which requires a new bloop, and it's struggling to shut down the old bloop server. this has been a recurring issue honestly. maybe it's time to really figure out how to fix it

**Comment by oyvindberg** [comment]
ok, I just disabled the auto-shutdown feature on windows (which was used to work around earlier problems). now it's running with old bleep/bloop throughout. good enough for now.

**Pull request merged**
